microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,96 @@
+
+#### Regional format
+To reflect a user's regional preferences for how data is displayed (such as the date), Microsoft Edge has a new option  in the **Language** section of **Settings**.
+
+![The 'Share additional OS regional format' menu with the Limited (default) option selected](media/os-regional-setting-screenshot.png)
+
+This access will allow websites to have visibility into more user's specific regional preferences than was previously possible. Users that change their OS regional preferences may now see their preferences reflected automatically in more websites.
+
+<!-- ====================================================================== -->
+## OS Regional Data Display Codes
+
+The user's formatting preferences are abstracted into a simple regional code, which is used by web applications to infer the various details noted above about dates, times, and other formatting preferences.
+
+Websites can use the built-in Internationalization API with a regional code to properly format information into the user's preferences.  The Internationalization API can save lots of MB of data transfer over the network, keeping websites small and allowing them to quickly and easily adapt to different regions.
+
+In general, the website's internationalization user experiences are only as good as the regional code that they're able to determine for the current user. 
+
+A regional code is a two-letter language code followed by a hyphen and a two-letter region code.  Microsoft Edge uses ICU to understand regional codes. ICU is an implementation of the Unicode standard's Internationalization API.  See [ICU](https://icu.unicode.org/home).
+
+To set the user's language preferences, Microsoft Edge gets the default values from the OS when Microsoft Edge is first installed. The user can then change the language preferences in `edge://settings/languages` and sort them in priority order.
+
+| Language or region data | User impact |
+|--------|-------------------------|
+| **Preferred language** in the **setting** page | The language selected for rendering Microsoft Edge. This language/region is also used for formatting values such as date, time, and numbers. |
+|  `accept-language` HTTP header | A list of regional codes sent to the server to allow a Website to display the right language. Microsoft Edge sends the regional codes in the order specified in **edge://settings/languages**. |
+| `navigator.language` API (and similar) | Allow a Website to display the right language and regional formatting for user input and other content produced locally. |
+
+## Regional Data Display options
+Users have three options on Microsoft Edge to select the level of information their browser will share about their regions:
+
+* Never 
+* Limited (default).
+* Always (not recommended). 
+
+#### Never
```

Comment:
move section down below "Always" section for consistent sequence per UI

Reply:
fixed